config/sync/core.entity_view_display.node.stanford_opportunity.full.yml

```diff
@@ -0,0 +1,346 @@
+uuid: 6839c55e-a372-4242-8b54-441d4472c8cd
```

Comment:
I think we need to re-start this work. This "Full" configuration should not be added, and the changes to the "Default" are not visible because of the use of layout builder.

Can you clean this up and use layout builder settings on the "default" display to make any necessary changes? `rm config/sync/core.entity_view_display.node.stanford_opportunity.full.yml && git checkout 11.x -- config/sync && drush cim -y` will get you back to the start. If you need a zoom huddle, let me know.